### PR TITLE
Only allow certain domains to be used as the src attribute for iframes

### PR DESCRIPTION
**Pull request by Hamms**
https://codedotorg.atlassian.net/browse/FND-291

### CODE REVIEW

File: curriculumBuilder/settings.py [hunk 1]
```diff
@@ -421,26 +423,55 @@
...
+def iframe_filter_domain(name, value):
+    """
+    We use iframes in curriculum descriptions to embed Google Docs containing
+    the lesson plans. We want to continue to allow iframes in general, but
+    restrict to only allow google docs.
+
+    If in the future we want to expand our usage of iframes to include other use
+    cases, those should be simililarly captured here.
+    """
+    if name in ('width', 'height', 'frameborder', 'style'):
```
Comment (bethanyaconnor) [inline curriculumBuilder/settings.py]: For my own education, why is this necessary?

Comment (Hamms) [reply curriculumBuilder/settings.py]: this function will override the '*' allowed tags, so we have to make sure that the attributes we want to allow are explicitly allowed here
